### PR TITLE
[velero] Bump Velero version to v1.5.1

### PR DESCRIPTION
**Pull request by svcua**
#### Checklist
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[velero]`)
- [x] Close issue #147, #122

### CODE REVIEW

File: CODE-OF-CONDUCT.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ members of the project's leadership.
 
 ## Attribution
 
-This Code of Conduct is adapted from the [Contributor Covenant][homepage], version 1.4,
+This Code of Conduct is adapted from the [Contributor Covenant][homepage], version 1.5,
```
Comment (jenting) [inline CODE-OF-CONDUCT.md:70]: I think this one does not need to bump version.

File: charts/velero/crds/backups.yaml [hunk 2]
```diff
@@ -8,7 +8,6 @@ metadata:
     "helm.sh/hook": pre-install,pre-upgrade
     "helm.sh/hook-delete-policy": "before-hook-creation"
     controller-gen.kubebuilder.io/version: v0.2.4
```
Comment (jenting) [inline charts/velero/crds/backups.yaml:10]: I think we should change the `controller-gen.kubebuilder.io/version` to `v0.3.0`, same as velero v1.5.1 release [1].

File: charts/velero/crds/backupstoragelocations.yaml [hunk 3]
```diff
@@ -8,17 +8,32 @@ metadata:
     "helm.sh/hook": pre-install,pre-upgrade
     "helm.sh/hook-delete-policy": "before-hook-creation"
     controller-gen.kubebuilder.io/version: v0.2.4
```
Comment (jenting) [inline charts/velero/crds/backupstoragelocations.yaml:10]: [1]

File: charts/velero/crds/deletebackuprequests.yaml [hunk 4]
```diff
@@ -8,7 +8,6 @@ metadata:
     "helm.sh/hook": pre-install,pre-upgrade
     "helm.sh/hook-delete-policy": "before-hook-creation"
     controller-gen.kubebuilder.io/version: v0.2.4
```
Comment (jenting) [inline charts/velero/crds/deletebackuprequests.yaml:10]: [1]

File: charts/velero/crds/downloadrequests.yaml [hunk 5]
```diff
@@ -8,7 +8,6 @@ metadata:
     "helm.sh/hook": pre-install,pre-upgrade
     "helm.sh/hook-delete-policy": "before-hook-creation"
     controller-gen.kubebuilder.io/version: v0.2.4
```
Comment (jenting) [inline charts/velero/crds/downloadrequests.yaml:10]: [1]

File: charts/velero/crds/podvolumebackups.yaml [hunk 6]
```diff
@@ -8,7 +8,6 @@ metadata:
     "helm.sh/hook": pre-install,pre-upgrade
     "helm.sh/hook-delete-policy": "before-hook-creation"
     controller-gen.kubebuilder.io/version: v0.2.4
```
Comment (jenting) [inline charts/velero/crds/podvolumebackups.yaml:10]: [1]

File: charts/velero/crds/podvolumerestores.yaml [hunk 7]
```diff
@@ -8,7 +8,6 @@ metadata:
     "helm.sh/hook": pre-install,pre-upgrade
     "helm.sh/hook-delete-policy": "before-hook-creation"
     controller-gen.kubebuilder.io/version: v0.2.4
```
Comment (jenting) [inline charts/velero/crds/podvolumerestores.yaml:10]: [1]

File: charts/velero/crds/resticrepositories.yaml [hunk 8]
```diff
@@ -8,7 +8,6 @@ metadata:
     "helm.sh/hook": pre-install,pre-upgrade
     "helm.sh/hook-delete-policy": "before-hook-creation"
     controller-gen.kubebuilder.io/version: v0.2.4
```
Comment (jenting) [inline charts/velero/crds/resticrepositories.yaml:10]: [1]

File: charts/velero/crds/restores.yaml [hunk 9]
```diff
@@ -8,7 +8,6 @@ metadata:
     "helm.sh/hook": pre-install,pre-upgrade
     "helm.sh/hook-delete-policy": "before-hook-creation"
     controller-gen.kubebuilder.io/version: v0.2.4
```
Comment (jenting) [inline charts/velero/crds/restores.yaml:10]: [1]

File: charts/velero/crds/schedules.yaml [hunk 10]
```diff
@@ -8,7 +8,6 @@ metadata:
     "helm.sh/hook": pre-install,post-upgrade
     "helm.sh/hook-delete-policy": "before-hook-creation"
     controller-gen.kubebuilder.io/version: v0.2.4
```
Comment (jenting) [inline charts/velero/crds/schedules.yaml:10]: [1]

File: charts/velero/crds/volumesnapshotlocations.yaml [hunk 11]
```diff
@@ -8,7 +8,6 @@ metadata:
     "helm.sh/hook": pre-install,pre-upgrade
     "helm.sh/hook-delete-policy": "before-hook-creation"
     controller-gen.kubebuilder.io/version: v0.2.4
```
Comment (jenting) [inline charts/velero/crds/volumesnapshotlocations.yaml:10]: [1]

**Comment by jenting** [comment]
Thanks @svcua for your contribution, super fast!

**Comment by stevehipwell** [comment]
@svcua I'm not familiar with the development of this project, but as a downstream chart consumer I'm curious why the upgrade from Velero v1.4 -> v1.5 is only considered as a patch upgrade?

**Comment by jenting** [comment]
It also fixes #122.

**Review by jenting — DISMISSED** [review]
LGTM, thanks @svcua 🚀

**Comment by svcua** [comment]
@carlisia @ashish-amarnath @nrb @cpanato PTAL

File: charts/velero/Chart.yaml [hunk 12]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
-appVersion: 1.4.2
+appVersion: 1.5.1
 description: A Helm chart for velero
 name: velero
-version: 2.12.17
+version: 2.12.18
```
Comment (cpanato) [inline charts/velero/Chart.yaml:5]: should be 
```suggestion
version: 2.13.0
```

Comment (cpanato) [reply charts/velero/Chart.yaml:5]: it is a minor change in the app, should be reflected in the chart as well.

There is any breaking change? if a user has the previous version and upgrades to this one, everything still works?

Comment (bclermont) [reply charts/velero/Chart.yaml:5]: If this is the last thing to do, can you please @svcua do this? to let every of us enjoy 1.5 :)

**Comment by bclermont** [comment]
been running this PR, after bumping it to [`2.13.0`](https://github.com/robotinfra/velero-helm-charts/commit/d67ca286459b22f6af0e681f66e7cc9c308d25da) and it works for me

**Review by jenting — APPROVED** [review]

**Review by jenting — APPROVED** [review]
LGTM, thanks ^^

**Review by cpanato — APPROVED** [review]
lgtm

**Comment by cpanato** [comment]
thanks so much for this PR!

**Pull request merged**